src/main/webapp/game.js

```diff
@@ -60,15 +64,47 @@ channel.bind(PUSHER_CHAT_CHANNEL_NAME, function(data) {
 });
 
 function embedPlaylist() {
-  fetch('/game').then(response => response.json()).then((videoID) => {
-    document.getElementById("player").src = "https://www.youtube.com/embed/" + videoID;
+  fetch('/game').then(response => response.json()).then((videoIdResponse) => {
+    videoId = videoIdResponse;
+
+    var tag = document.createElement('script');
+    var firstScript = document.getElementsByTagName('script')[0];
+
+    tag.src = 'https://www.youtube.com/iframe_api';
+    firstScript.parentNode.insertBefore(tag, firstScript);
+
+    document.getElementById("player").src = EMBED_URL + videoId + LOOP_PARAMS + videoId + VIDEO_PARAMS;
```

Comment:
I think it's actually less readable to use these constants here instead of the string literals, since you have to keep going between this line and the constants at the top of the class to understand what to full URL looks like.

I know it's often recommended to use constants instead of literals, but I think which one you should use depends on the situation. For example, having a constant called "ROUND_LENGTH_IN_SECONDS" is much better than using the literal "60" since it's not obvious what "60" means. However, it's pretty obvious what "&enablejsapi=1&autoplay=1&controls=0&modestbranding=1&disablekb=1&origin=http://localhost:8282" is.

Could you replace the constants with string literals?

Reply:
in this case I would definitely recommend literals here. For example a side effect now is that LOOP_PARAMS has to come first, before VIDEO_PARAMS since it contains the initial `?` however it is not obvious from looking at the code here that that is the case.